include/nanorange/algorithm/stable_partition.hpp

```diff
@@ -33,38 +34,39 @@ namespace detail {
 struct stable_partition_fn {
 private:
     template <typename I, typename Buf, typename Pred, typename Proj>
-    static I impl_buffered(I first, I last, Buf& buf, Pred& pred, Proj& proj)
+    static subrange<I> impl_buffered(I first, I last, Buf& buf, Pred& pred, Proj& proj)
```

Comment:
I'm still not sure I understand the reason for the changes in this function -- isn't changing the return value sufficient?